docs-release-notes.md

```diff
@@ -1,3 +1,20 @@
+# September 1, 2023
+
+## Triggers
+
+- Database Triggers: Add supported version information for document preimages on sharded clusters.
+
+## Reference
+
+- App Configuration Files
+  - GraphQL: Add `disable_schema_introspection` to the config object, clarify details about `use_natural_pluralization`.
+- App Services Admin API: Update documentation for the following endpoints:
+  - GraphQL API
+    - Get GraphQL API Configuration (GraphQL API GET and PUT): Add `disable_schema_introspection` to the config object, clarify details about `use_natural_pluralization`.
```

Comment:
Similar 
```suggestion
    - Get GraphQL API Configuration (GraphQL API GET and PUT): Add `disable_schema_introspection` to the config object. Clarify details about `use_natural_pluralization`.
```
Suggestion: Separate different tasks into sentences when needed since we now use periods.